### ISSUE TITLE
Why multiline values restart key-spacing alignment?

### ISSUE THREAD

**Issue by folmert** [eslint/eslint#9849]
My issue resonates from the fact that `key-spacing: ['error', {"align": "value"}]` rule is not compliant with Webstorm's "align object on value" option (which can be used to automatically reformat the code).

At first I thought it's a Webstorm issue and I reported it to JetBrains https://youtrack.jetbrains.com/issue/WEB-29680, but it appears that this behavior is not explicitly mentioned in ESLint docs. It's only mentioned in the code comment "Non-consecutive lines form a new group", but I'm not sure if it relates to this unjustified behavior.

**What rule do you want to change?**
key-spacing, "align": "value"
https://eslint.org/docs/2.0.0/rules/key-spacing

**Does this change cause the rule to produce more or fewer warnings?**
It would create less errors for Webstorm users.

**How will the change be implemented? (New option, new default behavior, etc.)?**
Either:

1. Ideally create a new option, with simplified aligning which doesn't form a new group after multiline values, or...
2. Explicitly mention in the docs that multiline values restart key-spacing alignment
 

**Please provide some example code that this change will affect:**

```js
var obj = {
        a:    value,
        bcde: 42,

        fg:   foo(), // Eslint error: Extra space before value
        h:    function () { // Eslint error: Extra space before value
            return this.a;
        },
        ijkl: 'Non-consecutive lines form a new group'
    };
```

**What does the rule currently do for this code?**
With mentioned rule turned on, above code is currently treated as incorrect ("Eslint error: Extra space before value"), and there is no rule that would accept this. 

**What will the rule do after it's changed?**
Instead of changing existing behavior (and confuse users that might find this current behavior useful), I'd rather add a new option for simplified aligning, that wouldn't throw an error in such case.

**Comment by folmert** [comment]
I've came up with a better, more minimal example (as previous one might be related with restarting alignment on new lines)

```javascript
var foo = {
    h:    function () { // Eslint error: Extra space before value
        return this.a;
    },
    ijkl: 'Non-consecutive lines form a new group',
};

var bar = {
    ijkl: 'Non-consecutive lines form a new group',
    h:    function () { // error is gone!
        return this.a;
    },
};
```

**Comment by pmcelhaney** [comment]
What version of ESLint do you have? It seems to work as expected on the latest (4.15.0).

[Demo](https://eslint.org/demo/#eyJ0ZXh0IjoidmFyIGZvbyA9IHtcclxuICAgIGg6ICAgIGZ1bmN0aW9uICgpIHsgLy8gRXNsaW50IGVycm9yOiBFeHRyYSBzcGFjZSBiZWZvcmUgdmFsdWVcclxuICAgICAgICByZXR1cm4gdGhpcy5hO1xyXG4gICAgfSxcclxuICAgIGlqa2w6ICdOb24tY29uc2VjdXRpdmUgbGluZXMgZm9ybSBhIG5ldyBncm91cCcsXHJcbn07XHJcblxyXG52YXIgYmFyID0ge1xyXG4gICAgaWprbDogJ05vbi1jb25zZWN1dGl2ZSBsaW5lcyBmb3JtIGEgbmV3IGdyb3VwJyxcclxuICAgIGg6ICAgIGZ1bmN0aW9uICgpIHsgLy8gZXJyb3IgaXMgZ29uZSFcclxuICAgICAgICByZXR1cm4gdGhpcy5hO1xyXG4gICAgfSxcclxufTsiLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJzY3JpcHQiLCJlY21hRmVhdHVyZXMiOnsianN4IjpmYWxzZSwiZ2xvYmFsUmV0dXJuIjpmYWxzZSwiaW1wbGllZFN0cmljdCI6ZmFsc2UsImV4cGVyaW1lbnRhbE9iamVjdFJlc3RTcHJlYWQiOmZhbHNlfX0sInJ1bGVzIjp7ImFjY2Vzc29yLXBhaXJzIjoib2ZmIiwiYXJyYXktYnJhY2tldC1uZXdsaW5lIjoib2ZmIiwiYXJyYXktYnJhY2tldC1zcGFjaW5nIjoib2ZmIiwiYXJyYXktY2FsbGJhY2stcmV0dXJuIjoib2ZmIiwiYXJyYXktZWxlbWVudC1uZXdsaW5lIjoib2ZmIiwiYXJyb3ctYm9keS1zdHlsZSI6Im9mZiIsImFycm93LXBhcmVucyI6Im9mZiIsImFycm93LXNwYWNpbmciOiJvZmYiLCJibG9jay1zY29wZWQtdmFyIjoib2ZmIiwiYmxvY2stc3BhY2luZyI6Im9mZiIsImJyYWNlLXN0eWxlIjoib2ZmIiwiY2FsbGJhY2stcmV0dXJuIjoib2ZmIiwiY2FtZWxjYXNlIjoib2ZmIiwiY2FwaXRhbGl6ZWQtY29tbWVudHMiOiJvZmYiLCJjbGFzcy1tZXRob2RzLXVzZS10aGlzIjoib2ZmIiwiY29tbWEtZGFuZ2xlIjoib2ZmIiwiY29tbWEtc3BhY2luZyI6Im9mZiIsImNvbW1hLXN0eWxlIjoib2ZmIiwiY29tcGxleGl0eSI6ImVycm9yIiwiY29tcHV0ZWQtcHJvcGVydHktc3BhY2luZyI6Im9mZiIsImNvbnNpc3RlbnQtcmV0dXJuIjoib2ZmIiwiY29uc2lzdGVudC10aGlzIjoib2ZmIiwiY29uc3RydWN0b3Itc3VwZXIiOiJlcnJvciIsImN1cmx5Ijoib2ZmIiwiZGVmYXVsdC1jYXNlIjoib2ZmIiwiZG90LWxvY2F0aW9uIjoib2ZmIiwiZG90LW5vdGF0aW9uIjoib2ZmIiwiZW9sLWxhc3QiOiJvZmYiLCJlcWVxZXEiOiJvZmYiLCJmb3ItZGlyZWN0aW9uIjoib2ZmIiwiZnVuYy1jYWxsLXNwYWNpbmciOiJvZmYiLCJmdW5jLW5hbWUtbWF0Y2hpbmciOiJvZmYiLCJmdW5jLW5hbWVzIjoib2ZmIiwiZnVuYy1zdHlsZSI6Im9mZiIsImdlbmVyYXRvci1zdGFyLXNwYWNpbmciOiJvZmYiLCJnZXR0ZXItcmV0dXJuIjoib2ZmIiwiZ2xvYmFsLXJlcXVpcmUiOiJvZmYiLCJndWFyZC1mb3ItaW4iOiJvZmYiLCJoYW5kbGUtY2FsbGJhY2stZXJyIjoib2ZmIiwiaWQtYmxhY2tsaXN0Ijoib2ZmIiwiaWQtbGVuZ3RoIjoib2ZmIiwiaWQtbWF0Y2giOiJvZmYiLCJpbmRlbnQtbGVnYWN5Ijoib2ZmIiwiaW5kZW50Ijoib2ZmIiwiaW5pdC1kZWNsYXJhdGlvbnMiOiJvZmYiLCJqc3gtcXVvdGVzIjoib2ZmIiwia2V5LXNwYWNpbmciOjIsImtleXdvcmQtc3BhY2luZyI6Im9mZiIsImxpbmUtY29tbWVudC1wb3NpdGlvbiI6Im9mZiIsImxpbmVicmVhay1zdHlsZSI6Im9mZiIsImxpbmVzLWFyb3VuZC1jb21tZW50Ijoib2ZmIiwibGluZXMtYXJvdW5kLWRpcmVjdGl2ZSI6Im9mZiIsIm1heC1kZXB0aCI6Im9mZiIsIm1heC1sZW4iOiJvZmYiLCJtYXgtbGluZXMiOiJvZmYiLCJtYXgtbmVzdGVkLWNhbGxiYWNrcyI6Im9mZiIsIm1heC1wYXJhbXMiOiJvZmYiLCJtYXgtc3RhdGVtZW50cy1wZXItbGluZSI6Im9mZiIsIm1heC1zdGF0ZW1lbnRzIjoib2ZmIiwibXVsdGlsaW5lLXRlcm5hcnkiOiJvZmYiLCJuZXctY2FwIjoib2ZmIiwibmV3LXBhcmVucyI6Im9mZiIsIm5ld2xpbmUtYWZ0ZXItdmFyIjoib2ZmIiwibmV3bGluZS1iZWZvcmUtcmV0dXJuIjoib2ZmIiwibmV3bGluZS1wZXItY2hhaW5lZC1jYWxsIjoib2ZmIiwibm8tYWxlcnQiOiJvZmYiLCJuby1hcnJheS1jb25zdHJ1Y3RvciI6Im9mZiIsIm5vLWF3YWl0LWluLWxvb3AiOiJvZmYiLCJuby1iaXR3aXNlIjoib2ZmIiwibm8tYnVmZmVyLWNvbnN0cnVjdG9yIjoib2ZmIiwibm8tY2FsbGVyIjoib2ZmIiwibm8tY2FzZS1kZWNsYXJhdGlvbnMiOiJlcnJvciIsIm5vLWNhdGNoLXNoYWRvdyI6Im9mZiIsIm5vLWNsYXNzLWFzc2lnbiI6ImVycm9yIiwibm8tY29tcGFyZS1uZWctemVybyI6ImVycm9yIiwibm8tY29uZC1hc3NpZ24iOiJlcnJvciIsIm5vLWNvbmZ1c2luZy1hcnJvdyI6Im9mZiIsIm5vLWNvbnNvbGUiOiJlcnJvciIsIm5vLWNvbnN0LWFzc2lnbiI6ImVycm9yIiwibm8tY29uc3RhbnQtY29uZGl0aW9uIjoiZXJyb3IiLCJuby1jb250aW51ZSI6Im9mZiIsIm5vLWNvbnRyb2wtcmVnZXgiOiJlcnJvciIsIm5vLWRlYnVnZ2VyIjoiZXJyb3IiLCJuby1kZWxldGUtdmFyIjoiZXJyb3IiLCJuby1kaXYtcmVnZXgiOiJvZmYiLCJuby1kdXBlLWFyZ3MiOiJlcnJvciIsIm5vLWR1cGUtY2xhc3MtbWVtYmVycyI6ImVycm9yIiwibm8tZHVwZS1rZXlzIjoiZXJyb3IiLCJuby1kdXBsaWNhdGUtY2FzZSI6ImVycm9yIiwibm8tZHVwbGljYXRlLWltcG9ydHMiOiJvZmYiLCJuby1lbHNlLXJldHVybiI6Im9mZiIsIm5vLWVtcHR5LWNoYXJhY3Rlci1jbGFzcyI6ImVycm9yIiwibm8tZW1wdHktZnVuY3Rpb24iOiJvZmYiLCJuby1lbXB0eS1wYXR0ZXJuIjoiZXJyb3IiLCJuby1lbXB0eSI6ImVycm9yIiwibm8tZXEtbnVsbCI6Im9mZiIsIm5vLWV2YWwiOiJvZmYiLCJuby1leC1hc3NpZ24iOiJlcnJvciIsIm5vLWV4dGVuZC1uYXRpdmUiOiJvZmYiLCJuby1leHRyYS1iaW5kIjoib2ZmIiwibm8tZXh0cmEtYm9vbGVhbi1jYXN0IjoiZXJyb3IiLCJuby1leHRyYS1sYWJlbCI6Im9mZiIsIm5vLWV4dHJhLXBhcmVucyI6Im9mZiIsIm5vLWV4dHJhLXNlbWkiOiJlcnJvciIsIm5vLWZhbGx0aHJvdWdoIjoiZXJyb3IiLCJuby1mbG9hdGluZy1kZWNpbWFsIjoib2ZmIiwibm8tZnVuYy1hc3NpZ24iOiJlcnJvciIsIm5vLWdsb2JhbC1hc3NpZ24iOiJlcnJvciIsIm5vLWltcGxpY2l0LWNvZXJjaW9uIjoib2ZmIiwibm8taW1wbGljaXQtZ2xvYmFscyI6Im9mZiIsIm5vLWltcGxpZWQtZXZhbCI6Im9mZiIsIm5vLWlubGluZS1jb21tZW50cyI6Im9mZiIsIm5vLWlubmVyLWRlY2xhcmF0aW9ucyI6ImVycm9yIiwibm8taW52YWxpZC1yZWdleHAiOiJlcnJvciIsIm5vLWludmFsaWQtdGhpcyI6Im9mZiIsIm5vLWlycmVndWxhci13aGl0ZXNwYWNlIjoiZXJyb3IiLCJuby1pdGVyYXRvciI6Im9mZiIsIm5vLWxhYmVsLXZhciI6Im9mZiIsIm5vLWxhYmVscyI6Im9mZiIsIm5vLWxvbmUtYmxvY2tzIjoib2ZmIiwibm8tbG9uZWx5LWlmIjoib2ZmIiwibm8tbG9vcC1mdW5jIjoib2ZmIiwibm8tbWFnaWMtbnVtYmVycyI6Im9mZiIsIm5vLW1peGVkLW9wZXJhdG9ycyI6Im9mZiIsIm5vLW1peGVkLXJlcXVpcmVzIjoib2ZmIiwibm8tbWl4ZWQtc3BhY2VzLWFuZC10YWJzIjoiZXJyb3IiLCJuby1tdWx0aS1hc3NpZ24iOiJvZmYiLCJuby1tdWx0aS1zcGFjZXMiOiJvZmYiLCJuby1tdWx0aS1zdHIiOiJvZmYiLCJuby1tdWx0aXBsZS1lbXB0eS1saW5lcyI6Im9mZiIsIm5vLW5hdGl2ZS1yZWFzc2lnbiI6Im9mZiIsIm5vLW5lZ2F0ZWQtY29uZGl0aW9uIjoib2ZmIiwibm8tbmVnYXRlZC1pbi1saHMiOiJvZmYiLCJuby1uZXN0ZWQtdGVybmFyeSI6Im9mZiIsIm5vLW5ldy1mdW5jIjoib2ZmIiwibm8tbmV3LW9iamVjdCI6Im9mZiIsIm5vLW5ldy1yZXF1aXJlIjoib2ZmIiwibm8tbmV3LXN5bWJvbCI6ImVycm9yIiwibm8tbmV3LXdyYXBwZXJzIjoib2ZmIiwibm8tbmV3Ijoib2ZmIiwibm8tb2JqLWNhbGxzIjoiZXJyb3IiLCJuby1vY3RhbC1lc2NhcGUiOiJvZmYiLCJuby1vY3RhbCI6ImVycm9yIiwibm8tcGFyYW0tcmVhc3NpZ24iOiJvZmYiLCJuby1wYXRoLWNvbmNhdCI6Im9mZiIsIm5vLXBsdXNwbHVzIjoib2ZmIiwibm8tcHJvY2Vzcy1lbnYiOiJvZmYiLCJuby1wcm9jZXNzLWV4aXQiOiJvZmYiLCJuby1wcm90byI6Im9mZiIsIm5vLXByb3RvdHlwZS1idWlsdGlucyI6Im9mZiIsIm5vLXJlZGVjbGFyZSI6ImVycm9yIiwibm8tcmVnZXgtc3BhY2VzIjoiZXJyb3IiLCJuby1yZXN0cmljdGVkLWdsb2JhbHMiOiJvZmYiLCJuby1yZXN0cmljdGVkLWltcG9ydHMiOiJvZmYiLCJuby1yZXN0cmljdGVkLW1vZHVsZXMiOiJvZmYiLCJuby1yZXN0cmljdGVkLXByb3BlcnRpZXMiOiJvZmYiLCJuby1yZXN0cmljdGVkLXN5bnRheCI6Im9mZiIsIm5vLXJldHVybi1hc3NpZ24iOiJvZmYiLCJuby1yZXR1cm4tYXdhaXQiOiJvZmYiLCJuby1zY3JpcHQtdXJsIjoib2ZmIiwibm8tc2VsZi1hc3NpZ24iOiJlcnJvciIsIm5vLXNlbGYtY29tcGFyZSI6Im9mZiIsIm5vLXNlcXVlbmNlcyI6Im9mZiIsIm5vLXNoYWRvdy1yZXN0cmljdGVkLW5hbWVzIjoib2ZmIiwibm8tc2hhZG93Ijoib2ZmIiwibm8tc3BhY2VkLWZ1bmMiOiJvZmYiLCJuby1zcGFyc2UtYXJyYXlzIjoiZXJyb3IiLCJuby1zeW5jIjoib2ZmIiwibm8tdGFicyI6Im9mZiIsIm5vLXRlbXBsYXRlLWN1cmx5LWluLXN0cmluZyI6Im9mZiIsIm5vLXRlcm5hcnkiOiJvZmYiLCJuby10aGlzLWJlZm9yZS1zdXBlciI6ImVycm9yIiwibm8tdGhyb3ctbGl0ZXJhbCI6Im9mZiIsIm5vLXRyYWlsaW5nLXNwYWNlcyI6Im9mZiIsIm5vLXVuZGVmLWluaXQiOiJvZmYiLCJuby11bmRlZiI6ImVycm9yIiwibm8tdW5kZWZpbmVkIjoib2ZmIiwibm8tdW5kZXJzY29yZS1kYW5nbGUiOiJvZmYiLCJuby11bmV4cGVjdGVkLW11bHRpbGluZSI6ImVycm9yIiwibm8tdW5tb2RpZmllZC1sb29wLWNvbmRpdGlvbiI6Im9mZiIsIm5vLXVubmVlZGVkLXRlcm5hcnkiOiJvZmYiLCJuby11bnJlYWNoYWJsZSI6ImVycm9yIiwibm8tdW5zYWZlLWZpbmFsbHkiOiJlcnJvciIsIm5vLXVuc2FmZS1uZWdhdGlvbiI6ImVycm9yIiwibm8tdW51c2VkLWV4cHJlc3Npb25zIjoib2ZmIiwibm8tdW51c2VkLWxhYmVscyI6ImVycm9yIiwibm8tdXNlLWJlZm9yZS1kZWZpbmUiOiJvZmYiLCJuby11c2VsZXNzLWNhbGwiOiJvZmYiLCJuby11c2VsZXNzLWNvbXB1dGVkLWtleSI6Im9mZiIsIm5vLXVzZWxlc3MtY29uY2F0Ijoib2ZmIiwibm8tdXNlbGVzcy1jb25zdHJ1Y3RvciI6Im9mZiIsIm5vLXVzZWxlc3MtZXNjYXBlIjoiZXJyb3IiLCJuby11c2VsZXNzLXJlbmFtZSI6Im9mZiIsIm5vLXVzZWxlc3MtcmV0dXJuIjoib2ZmIiwibm8tdmFyIjoib2ZmIiwibm8tdm9pZCI6Im9mZiIsIm5vLXdhcm5pbmctY29tbWVudHMiOiJvZmYiLCJuby13aGl0ZXNwYWNlLWJlZm9yZS1wcm9wZXJ0eSI6Im9mZiIsIm5vLXdpdGgiOiJvZmYiLCJub25ibG9jay1zdGF0ZW1lbnQtYm9keS1wb3NpdGlvbiI6Im9mZiIsIm9iamVjdC1jdXJseS1uZXdsaW5lIjoib2ZmIiwib2JqZWN0LWN1cmx5LXNwYWNpbmciOiJlcnJvciIsIm9iamVjdC1wcm9wZXJ0eS1uZXdsaW5lIjoib2ZmIiwib2JqZWN0LXNob3J0aGFuZCI6Im9mZiIsIm9uZS12YXItZGVjbGFyYXRpb24tcGVyLWxpbmUiOiJvZmYiLCJvbmUtdmFyIjoib2ZmIiwib3BlcmF0b3ItYXNzaWdubWVudCI6Im9mZiIsIm9wZXJhdG9yLWxpbmVicmVhayI6Im9mZiIsInBhZGRlZC1ibG9ja3MiOiJvZmYiLCJwYWRkaW5nLWxpbmUtYmV0d2Vlbi1zdGF0ZW1lbnRzIjoib2ZmIiwicHJlZmVyLWFycm93LWNhbGxiYWNrIjoib2ZmIiwicHJlZmVyLWNvbnN0Ijoib2ZmIiwicHJlZmVyLWRlc3RydWN0dXJpbmciOiJvZmYiLCJwcmVmZXItbnVtZXJpYy1saXRlcmFscyI6Im9mZiIsInByZWZlci1wcm9taXNlLXJlamVjdC1lcnJvcnMiOiJvZmYiLCJwcmVmZXItcmVmbGVjdCI6Im9mZiIsInByZWZlci1yZXN0LXBhcmFtcyI6Im9mZiIsInByZWZlci1zcHJlYWQiOiJvZmYiLCJwcmVmZXItdGVtcGxhdGUiOiJvZmYiLCJxdW90ZS1wcm9wcyI6Im9mZiIsInF1b3RlcyI6Im9mZiIsInJhZGl4Ijoib2ZmIiwicmVxdWlyZS1hd2FpdCI6Im9mZiIsInJlcXVpcmUtanNkb2MiOiJvZmYiLCJyZXF1aXJlLXlpZWxkIjoiZXJyb3IiLCJyZXN0LXNwcmVhZC1zcGFjaW5nIjoib2ZmIiwic2VtaS1zcGFjaW5nIjoib2ZmIiwic2VtaS1zdHlsZSI6Im9mZiIsInNlbWkiOiJvZmYiLCJzb3J0LWltcG9ydHMiOiJvZmYiLCJzb3J0LWtleXMiOiJvZmYiLCJzb3J0LXZhcnMiOiJvZmYiLCJzcGFjZS1iZWZvcmUtYmxvY2tzIjoib2ZmIiwic3BhY2UtYmVmb3JlLWZ1bmN0aW9uLXBhcmVuIjoib2ZmIiwic3BhY2UtaW4tcGFyZW5zIjoib2ZmIiwic3BhY2UtaW5maXgtb3BzIjoib2ZmIiwic3BhY2UtdW5hcnktb3BzIjoib2ZmIiwic3BhY2VkLWNvbW1lbnQiOiJvZmYiLCJzdHJpY3QiOiJvZmYiLCJzd2l0Y2gtY29sb24tc3BhY2luZyI6Im9mZiIsInN5bWJvbC1kZXNjcmlwdGlvbiI6Im9mZiIsInRlbXBsYXRlLWN1cmx5LXNwYWNpbmciOiJvZmYiLCJ0ZW1wbGF0ZS10YWctc3BhY2luZyI6Im9mZiIsInVuaWNvZGUtYm9tIjoib2ZmIiwidXNlLWlzbmFuIjoiZXJyb3IiLCJ2YWxpZC1qc2RvYyI6Im9mZiIsInZhbGlkLXR5cGVvZiI6ImVycm9yIiwidmFycy1vbi10b3AiOiJvZmYiLCJ3cmFwLWlpZmUiOiJvZmYiLCJ3cmFwLXJlZ2V4Ijoib2ZmIiwieWllbGQtc3Rhci1zcGFjaW5nIjoib2ZmIiwieW9kYSI6Im9mZiJ9LCJlbnYiOnsiYnJvd3NlciI6ZmFsc2UsIm5vZGUiOmZhbHNlLCJjb21tb25qcyI6ZmFsc2UsInNoYXJlZC1ub2RlLWJyb3dzZXIiOmZhbHNlLCJ3b3JrZXIiOmZhbHNlLCJhbWQiOmZhbHNlLCJtb2NoYSI6ZmFsc2UsImphc21pbmUiOmZhbHNlLCJqZXN0IjpmYWxzZSwicGhhbnRvbWpzIjpmYWxzZSwianF1ZXJ5IjpmYWxzZSwicXVuaXQiOmZhbHNlLCJwcm90b3R5cGVqcyI6ZmFsc2UsInNoZWxsanMiOmZhbHNlLCJtZXRlb3IiOmZhbHNlLCJtb25nbyI6ZmFsc2UsInByb3RyYWN0b3IiOmZhbHNlLCJhcHBsZXNjcmlwdCI6ZmFsc2UsIm5hc2hvcm4iOmZhbHNlLCJzZXJ2aWNld29ya2VyIjpmYWxzZSwiYXRvbXRlc3QiOmZhbHNlLCJlbWJlcnRlc3QiOmZhbHNlLCJ3ZWJleHRlbnNpb25zIjpmYWxzZSwiZXM2IjpmYWxzZSwiZ3JlYXNlbW9ua2V5IjpmYWxzZX19fQ==)

**Comment by folmert** [comment]
Also 4.15.0. For me in command line ESLint returns only first error, same for ESLint as PhpStorm's plugin. So it appears it's not only unexpected, but also bugged.
![alt text](https://i.imgur.com/dSHDO7l.jpg "ESLint bug")

I'd say it works as expected if it aligns on value always, regardless if the value is multiline or not.

**Comment by pmcelhaney** [comment]
Here's a minimal test case. Clone, install, and run, and see if you get the same result I did.

https://github.com/pmcelhaney/eslint-issue-9849

**Comment by folmert** [comment]
You're using different rule in `.eslintrc.js`:
```javascript
"key-spacing": [2, {"beforeColon": false, "afterColon": true}]
```

It should be:
```javascript
'key-spacing': ['error', {"align": "value"}]
```

**Comment by pmcelhaney** [comment]
Okay, I get it now. I agree, this is a bug. Simply swapping the order of properties shouldn't change the behavior.

**Comment by not-an-aardvark** [comment]
Thanks for your interest in improving ESLint. Unfortunately, it looks like this issue didn't get consensus from the team, so I'm closing it. We define [consensus](http://eslint.org/docs/maintainer-guide/issues.html#consensus) as having three :+1:s from team members, as well as a team member willing to champion the proposal. This is a high bar by design -- we can't realistically accept and maintain every feature request in the long term, so we only accept feature requests which are useful enough that there is consensus among the team that they're worth adding.

Since ESLint is pluggable and can load custom rules at runtime, the lack of consensus among the ESLint team doesn't need to be a blocker for you using this in your project, if you'd find it useful. It just means that you would need to [implement the rule](http://eslint.org/docs/developer-guide/working-with-rules) yourself, rather than using a bundled rule that is packaged with ESLint.
